src/UI/components/sources/source_form.tsx

```diff
@@ -0,0 +1,194 @@
+import { Paper, Button, Typography, Divider, Box, Grid } from '@mui/material';
+import { useForm, SubmitHandler, Controller } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormLabel,
+  FormControlLabel,
+  TextField,
+} from '@material-ui/core';
```

Comment:
We should be using the `@mui` package for these things - material-ui has been replaced my mui